src/components/ErrorBanner.js

```diff
@@ -27,23 +27,30 @@ class ErrorBanner extends Component {
       }
     },
     [
-      h(Clickable, {
-        disabled: onFirst,
-        style: { display: 'flex', color: onFirst ? colors.gray[2] : null },
-        onClick: () => this.setState({ errorNumber: errorNumber - 1 })
-      }, [icon('angle left')]),
       div({
         style: {
-          fontWeight: 500,
-          backgroundColor: 'rgba(255, 255, 255, 0.2', borderRadius: '1rem',
-          padding: '0.5rem 1rem', margin: '-0.25rem 0'
+          display: 'flex', alignItems: 'center', padding: '1rem', marginRight: '1rem',
```

Comment:
So I tried following your suggestion and added padding back into TopBanner like how it was before, but the issue is then we can't get the line to span over the banner. To combat that, we wrapped everything on the left side of the line into a div and gave it a border, then modified that div (via padding, etc.). It also happened to adjust the divs on the right side via flex alignment which, as Putnam puts, is quite precarious.

I could try to adjust it so the right side has matching style elements rather than just relying on flex?

Reply:
I don't like that TopBanner imposes a 1rem padding, then ErrorBanner cancels it via -1rem margin. That means that the line is only right because of dueling magic numbers. I think it should be up to the child to say how much padding they want. The parent could have an overridable default of padding: 1rem but should expose the ability to override that.

(I feel the same about our modals and various page wrappers as well)

Reply:
Added the parent default padding of 1rem that is overridden in ErrorBanner